df.history.xml

```diff
@@ -1422,6 +1422,7 @@
         <enum-item name='FALLING_OBJECT'/>
         <enum-item name='LEAPT_FROM_HEIGHT'/>
         <enum-item name='DROWN_ALT2'/>
+        <enum-item name='EXECUTION_GENERIC'/>
```

Comment:
Are you sure the item below is separate? Not sure when it was introduced.
(You can also use `since` for these, by the way, if you want)

Reply:
I'm not sure I understand the question. I named an element someone else had introduced as unnamed at some unspecified time (presumably together with the still unnamed one below it), so I can't apply a "since" to it. The name is applied to an observed number for a field of this type, and so isn't a separate term to group other kinds of executions. Rather, I assume it means executed without specifying the method.

Reply:
To clarify - from the diff, you added a new element here instead of renaming the item below it. (It has the effect you want, but adds an extra unnamed element to the enum. Was that intended?)

Reply:
![image](https://user-images.githubusercontent.com/3719547/75041110-e821bf80-5489-11ea-8fe8-83806c78ba3c.png)

Reply:
It that's the case, no. I was sure I modified one out of two no-name elements by adding a name section to it.